src/material-examples/BUILD.bazel

```diff
@@ -1,39 +1,72 @@
 package(default_visibility = ["//visibility:public"])
 
-load("//:packages.bzl", "CDK_EXPERIMENTAL_TARGETS", "CDK_TARGETS", "MATERIAL_EXPERIMENTAL_TARGETS", "MATERIAL_PACKAGES", "ROLLUP_GLOBALS")
+load("//:packages.bzl", "ROLLUP_GLOBALS")
 load("//tools:defaults.bzl", "ng_module", "ng_package")
 load("//tools/highlight-files:index.bzl", "highlight_files")
 load("//tools/package-docs-content:index.bzl", "package_docs_content")
 
+EXAMPLE_PACKAGES = [
+    # TODO(devversion): try to have for each entry-point a bazel package so that
+    # we can automate this using the "package.bzl" variables.
+    "//src/material-examples/cdk/a11y",
```

Comment:
Yes that was one of the ideas when I made separate directories :smile: It just involves a bit more work for now since our docs only expect one examples package right now.